src/Tickets/Commerce/Shortcodes/Checkout_Shortcode.php

```diff
@@ -60,15 +60,8 @@ public function setup_template_vars() {
 			'is_tec_active'      => defined( 'TRIBE_EVENTS_FILE' ) && class_exists( 'Tribe__Events__Main' ),
 			'gateways'           => $gateways,
 			'gateways_active'    => $this->get_gateways_active(),
-			'gateways_connected' => $this->get_gateways_connected(),
-			'gateways_enabled' => 0,
+			'gateways_connected' => $this->get_available_gateways(),
```

Comment:
Not sure if we can just completely remove a public method. It'd also be better if we're consistent with the naming. If we're using `get_gateways_*()`, let's keep on using that format instead of switching one to `get_*_gateways()`